server/server.go

```diff
@@ -47,8 +52,17 @@ func ListenAndServe(ctx context.Context, httpsPort, httpPort int, handler http.H
 		opts.TLSListenerConfig.TLSConfig = &tls.Config{}
 	}
 
-	logger := logrus.StandardLogger()
-	errorLog := log.New(logger.WriterLevel(logrus.DebugLevel), "", log.LstdFlags)
+	writer := logrus.StandardLogger().WriterLevel(logrus.DebugLevel)
+	if opts.DisplayServerLogs {
+		// Create a new logger the writes to which are at the same level as it's
+		// configured at, with an "ERROR" prefix (as the server writes "un-levelled"
+		// text to the log stream it's handled).
+		logger := logrus.New()
```

Comment:
By using `logrus.New()` rather than `logrus.StandardLogger()` I think this could discard changes made to the original standard logger by another application (e.g. rancher). Since we don't do this in the old case, I'm not sure why we do this here - can you elaborate on why these behave differently?

I would propose that this mirror the old side:

```suggestion
		logger := logrus.StandardLogger()
```